tests/sentry/taskworker/test_config.py

```diff
@@ -18,6 +18,17 @@ def test_import_paths():
             raise AssertionError(f"Unable to import {path} from TASKWORKER_IMPORTS")
 
 
+def test_taskworker_schedule_unique() -> None:
+    visited = {}
+    for key, entry in settings.TASKWORKER_SCHEDULES.items():
+        if entry["task"] in visited:
+            msg = f"Schedule {key} references a task that is already scheduled: {entry['task']}"
+            raise AssertionError(msg)
+
+        if entry["task"] not in visited:
```

Comment:
This check is unnecessary.

Reply:
You're totally right. I missed this when removing a different approach 🤦